src/main/java/Entities/Board.java

```diff
@@ -1,16 +1,26 @@
 package Entities;
 
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
+/**
+ * Represents a Monopoly board.
+ */
 public class Board{
 
-    Player[] players;
-    Cell[] cells;
+    List<Player> players;
+    List<Cell> cells;
```

Comment:
Having board keeps track of cells and properties might be redundant since a cell might be a property or an action space already.